### ISSUE TITLE
NO PretrainDatasetPkl humanml3D_train/test.pkl

### ISSUE THREAD

**Issue by 41xu** [MotionXperts/MotionExpert#3]
when running pretrain stage, there's no humanml3D_train.pkl and no humanml3D_test.pkl

**Comment by weihsinyeh** [comment]
Yes, because both are the same as the datasets in HumanML3D train and test.
> https://github.com/EricGuo5513/HumanML3D

**Comment by 41xu** [comment]
ok, i'll try to convert new_joints file to h3d.pkl file. thanks for your prompt response

**Comment by 41xu** [comment]
can you give the data.keys() of humanml3d.pkl? is it same as your released file?
pkl: list of data;
each: ['coordinates': tensor, with the shape: [N, 66], 'labels': class of list, which is the text descriptions of humanml3d, 'video_name': filename of h3d motion]? or in which kind of format?

**Comment by weihsinyeh** [comment]
Yes.
The dataset is saved as a pickle file and is of type `<class 'list'>`. 
The type of each entry is `<class 'dict'>`.
Each entry in the dataset contains the following information :
video_name: `<class 'str'>`
coordinates: `<class 'torch.Tensor'>`
labels: `<class 'list'>`
```
{'video_name': '000001', 
'features': tensor([[ 0.0000,  1.0143,  0.0000,  ..., -0.3651,  0.9127, -0.1035],
        [ 0.0000,  1.0143,  0.0000,  ..., -0.3696,  0.8961, -0.1261],
        [ 0.0000,  1.0143,  0.0000,  ..., -0.3740,  0.8825, -0.1293],
        ...,
        [ 0.0000,  1.0143,  0.0000,  ..., -0.5194,  0.9955, -0.0585],
        [ 0.0000,  1.0143,  0.0000,  ..., -0.4970,  0.9767, -0.0515],
        [ 0.0000,  1.0143,  0.0000,  ..., -0.4630,  0.9588, -0.0415]]), 
'labels': ['a man squats extraordinarily low then bolts up in an unsatisfactory jump.', 'a person falls to the ground in a sitting motion and then pops back up in a standing position.', 'a person squats down then jumps', 'a descends into a falling motion and thens bounces back up.']}
```
In this case : the shape of coordinates is `torch.Size([35, 66])`

**Comment by 41xu** [comment]
thanks. now it works. btw i suggest you update the description of h3d.pkl data preparation.

**Comment by 41xu** [comment]
solid work and clear updates. now everything is very clear. thanks!
